doc/api/worker_threads.md

```diff
@@ -325,6 +432,32 @@ new Worker('process.env.SET_IN_WORKER = "foo"', { eval: true, env: SHARE_ENV })
   });
 ```
 
+## `worker.setConnectionsListener(fn)`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1.1 - Active development
+
+* `fn` {Function} A callback to be executed when [`worker.connect()`][] is called from another thread.
+  The function will receive the following arguments:
+
+  * `sender` {number} The other thread id.
+  * `port` {MessagePort} The port than can be used to communicate with the other thread.
+  * `data` {any} The data passed as second argument to [`worker.connect()`][].
+
+  The function must return `true` to accept the connection or any other value to
+  refuse the connection. If the function returns a `Promise`, it will be awaited.
```

Comment:
```suggestion
The function must return `true` to accept the connection or any other value to
refuse the connection. If the function returns a `Promise`, it will be awaited.
```

Reply:
Thanks, changed!